common/src/main/java/com/wynntils/models/beacons/type/BeaconColor.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.beacons.type;
+
+import java.util.List;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+import org.apache.commons.compress.utils.Lists;
+
+public enum BeaconColor {
+    YELLOW((int) (0.09375 * 32)),
+    BLUE((int) (0.125 * 32)),
+    PURPLE((int) (0.15625 * 32)),
+    GRAY((int) (0.1875 * 32)),
+    ORANGE((int) (0.21875 * 32)),
+    RED((int) (0.25 * 32)),
+    DARK_GRAY((int) (0.28125 * 32)),
+    WHITE((int) (0.3125 * 32)),
+    CYAN((int) (0.34375 * 32)),
+    RAINBOW((int) (0.375 * 32));
+
```

Comment:
Obviously I got this from the resource pack, but I wanted to keep the format (damagePredicate * itemDurability)